symfony.lock

```diff
@@ -420,7 +420,7 @@
         "version": "2.0.1"
     },
     "php": {
-        "version": "7.2"
+        "version": "7.3"
```

Comment:
Should this change be part of this PR?

Reply:
I guess this happened because my local environment was using PHP 7.3 while I developed that

Reply:
Fixed this